libs/components/phone-field/testing/src/phonefield/phone-field-fixture.spec.ts

```diff
@@ -72,18 +72,19 @@ const VALID_US_NUMBER = '8675555309';
   `,
 })
 class PhoneFieldTestComponent implements OnInit {
-  public allowExtensions: boolean = true;
-  public defaultCountry: string;
-  public disabled: boolean;
-  public noValidate: boolean = false;
-  public returnFormat: SkyPhoneFieldNumberReturnFormat;
-  public selectedCountry: SkyPhoneFieldCountry;
-  public showInvalidDirective: boolean = false;
-  public supportedCountryISOs: string[];
-
-  public phoneControl: UntypedFormControl;
-  public phoneForm: UntypedFormGroup;
-
+  public allowExtensions = true;
+  public defaultCountry: string | undefined;
+  public disabled: boolean | undefined;
+  public noValidate = false;
+  public returnFormat: SkyPhoneFieldNumberReturnFormat | undefined;
+  public selectedCountry: SkyPhoneFieldCountry | undefined;
+  public showInvalidDirective = false;
+  public supportedCountryISOs: string[] | undefined;
+
+  public phoneControl: UntypedFormControl | undefined;
+  public phoneForm: UntypedFormGroup | undefined;
+
+  // TODO: replace this with a fake spy
```

Comment:
this is just a dummy function that does not need to exist here. a fake Spy could replace it

Reply:
I'm not sure I understand? Can we remove/replace it now?

Reply:
The function does not need to exist. All it's being used for is confirming that it is called. Yes, we can replace it now.